scripts/auto_mesh.py

```diff
@@ -260,8 +262,8 @@ def query():
 
     # Get the ip addresses of the routers already running
     if ip_list:
-        si = ip_list.index(ip)
-        ip_list = ip_list[:si]
+        if ip in ip_list:
```

Comment:
Not sure this is equivalent? I believe previously it is keeping all the entries *before* the ip of the local pod. With the change it will include all ips except the local pod.

Reply:
Ok, maybe I misunderstood its purpose. Let me retry it with the original code and see.